transforms/dialog-full-screen-app-wrapper/dialog-full-screen-app-wrapper.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Add <AppWrapper> around the children on <DialogFullScreen>
+ * <DialogFullScreen><AppWrapper>{ children }</ AppWrapper></ DialogFullScreen>
+ */
+import registerMethods from "../registerMethods";
+
+function transformer(fileInfo, api, options) {
+  const j = api.jscodeshift;
+  registerMethods(j);
+
+  const addImport = () => {
+    // DialogFullScreen import
+    const dialogFullScreenImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/dialog-full-screen"
+      }
+    });
+
+    const appWrapperImportString = "carbon-react/lib/components/app-wrapper";
+
+    // AppWrapper import
+    let appWrapperImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: appWrapperImportString,
+      },
+    });
+
+    // if no AppWrapper import exists already add one
+    if (!appWrapperImport.size()) {
+      appWrapperImport = j.importDeclaration(
+        [j.importDefaultSpecifier(j.identifier("AppWrapper"))],
+        j.stringLiteral(appWrapperImportString)
+      );
+    
+      // insert AppWrapper import after DialogFullScreen import
+      dialogFullScreenImport.insertAfter(appWrapperImport);
+
+      return true;
+    } else {
```

Comment:
Do you need this else block? I think the function just returning undefined might be enough here